checks.d/mapreduce.py

```diff
@@ -127,8 +127,19 @@ def check(self, instance):
         if rm_address is None:
             raise Exception('The ResourceManager URL must be specified in the instance configuration')
 
-        # Get the cluster ID from Yarn
-        cluster_id = self._get_cluster_id(rm_address)
+        # Get additional tags from the conf file
+        tags = instance.get('tags', [])
+        if tags is None:
+            tags = []
+        else:
+            tags = list(set(tags))
+
+        # Get the cluster name from the conf file
+        cluster_name = instance.get('cluster_name')
+        if cluster_name is None:
+            raise Exception('The cluster_name must be specified in the instance configuration')
```

Comment:
same comment as the YARN check, it'd be better to add a warning and set a default value instead of raising an exception.